README.md

```diff
@@ -7,9 +7,10 @@ Decision Diagrams for Discrete Optimization in Julia.
 [![Build Status](https://travis-ci.com/rschwarz/Diderot.jl.svg?branch=master)](https://travis-ci.com/rschwarz/Diderot.jl)
 [![Codecov](https://codecov.io/gh/rschwarz/Diderot.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/rschwarz/Diderot.jl)
 
-Provides a generic implementation of decisision diagrams (top-down construction
+Provides a generic implementation of decision diagrams (top-down construction
 of layered state transition graph). Implements a branch-and-bound algorithms
-with subproblems defined by nodes in an exact cutset of the diagram.
+with subproblems defined by nodes in an exact
+[cutset](https://en.wikipedia.org/wiki/Cut_%28graph_theory%29) of the diagram.
```

Comment:
I appreciate the fix & link.

But I think that a cutset in this context is not quite the same as a cut in a graph. I mean, technically, it's a cut, but selecting a subset of the nodes. But, importantly, it selects a set of nodes such that any path from root to terminal goes through on of these nodes. So, it's more closely related to [vertex separators](https://en.wikipedia.org/wiki/Vertex_separator), but specifically for s-t paths in DAGs.

Reply:
Thanks, good to know, if I remember. ;) You can at least fix the typo and add a link if you want. I'm not pushing for that, I know a bit about graphs and cuts, but clearly not enough. So I went googling and since I found it there in bold (and I don't see in anywhere in the other article), you be the judge if a link (somewhere) is helpful.

Reply:
Yeah, I see the definition on Wikipedia, but it references edges, not nodes.